lib/vagrant-service-manager/services/open_shift.rb

```diff
@@ -10,12 +10,12 @@ def initialize(machine, ui)
       def execute
         full_cmd = "#{@extra_cmd} sccli openshift"
 
-        @machine.communicate.sudo(full_cmd) do |type, data|
-          if type == :stderr
-            @ui.error(data)
-            exit 126
-          end
+        exit_code = @machine.communicate.sudo(full_cmd)
+        unless exit_code.zero?
+          @env.ui.error "#{full_cmd} failed"
+          exit 126
```

Comment:
Should we not print the error output when exit code is non-zero? We should pass the failure output to users.